scripts/setup_integration/docker-compose.redis.yml

```diff
@@ -0,0 +1,6 @@
+services:
+  redis:
+    image: redis
```

Comment:
Unrelated to your change... I feel like we _may_ want to specify a tag?

Reply:
we should, yes!